tests/rptest/clients/kubectl.py

```diff
@@ -165,7 +165,21 @@ def _cmd(self, cmd):
                 f'--------- stderr -----------\n{e.stderr.decode()}')
             raise
 
-    def cmd(self, kcmd: list[str] | str):
+    def _cmd_capture(self, cmd):
+        # Log and run
+        ssh_prefix = self._ssh_prefix()
+        remote_cmd = ssh_prefix + cmd
+        self._redpanda.logger.info(remote_cmd)
+        try:
+            process = subprocess.Popen(remote_cmd,
+                                       stdout=subprocess.PIPE,
+                                       stderr=subprocess.STDOUT)
+            for line in process.stdout:  # type: ignore
+                yield line
+        except Exception as e:
+            raise RuntimeError(f'Failed to capture output') from e
```

Comment:
What value does this exception chaining add? Actually I find the error confusing because it's not really the capture that failed in almost all cases but the process itself.